morpheushelper/cogs/wikipedia.py

```diff
@@ -0,0 +1,59 @@
+from discord import Member
+from discord.embeds import Embed
+from discord.ext import commands
+from discord.ext.commands import Cog, Bot
+from discord.ext.commands.context import Context
+from PyDrocsid.async_thread import run_in_thread
+from PyDrocsid.translations import translations
+from wikipedia.exceptions import WikipediaException, DisambiguationError, PageError
+from colours import Colours
+import wikipedia
+
+
+def make_embed(title: str, content: str, color, requested_by: Member) -> Embed:
+    embed = Embed(title=title, description=content, color=color)
+    embed.set_footer(text=translations.f_requested_by(requested_by, requested_by.id), icon_url=requested_by.avatar_url)
+    return embed
+
+
+# Note: wikipedia allows bots, but only bots that are responsible (not going too fast).
+class WikipediaCog(Cog, name="Wikipedia"):
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    @commands.command(aliases=["wiki", "summary", "wk", "sum"])
+    @commands.cooldown(5, 10, commands.BucketType.user)
+    async def wikipedia(self, ctx: Context, *, title: str):
+        """
+        display wikipedia summary about a topic
+        """
+
+        # workaround because the run_in_thread function doesn't support arguments
+        def inner():
+            try:
+                return [wikipedia.summary(title)]
+
+            except DisambiguationError as e:
+                return {"Error": str(e)}
+
+            except PageError as e:
+                return str(e)
+
+            except WikipediaException:
+                return "Wikipedia cog is not available currently!"
+
+        summary = await run_in_thread(inner)
```

Comment:
I'm sry for your inconvenience but after seeing your problem with the `run_in_thread` method I updated the function. It's now possible to hand over args and kwargs to the `run_in_thread` method and handling the exception in the main thread.

so it should bei possible for you to rewrite your code like this
```python
try:
    summary = await run_in_thread(wikipedia.summary, title)
except PageError as e:
    pass
```